### PR TITLE
Enable Jira Data Center integration doc migrated

### PR DESCRIPTION
**Pull request by Hikkigaya**
Enable Jira Data Center integration doc migrated along with images, nav updated

### Summary

Enable Jira Data Center integration doc migrated along with images, nav updated
Preview link: http://10.2.7.208/integrations/jira/enable-jira-data-center-integration

### Related PRs, issues, or features (optional)

- 234

### Metadata

- [x] Adds new file(s)
- [x] Edits existing file(s)
- [ ] Removes file(s)

### PR contributor checklist

- [x] My PR follows the [Kobiton Docs contributor guidelines](https://github.com/kobiton/docs/blob/main/CONTRIBUTE.adoc), meaning:

    - My content contains correct spelling and grammar.
    - My directories and files are [named](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-and-file-names) and [structured](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-structure) correctly.
    - My style and voice follows the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human).
    - I added all new pages to their section's [`nav.adoc` file](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#configure-navigation-in-navadoc).
    - I removed all localizations (like `en-us`) from my URLs.

### CODE REVIEW

**Review by tungmhoang — CHANGES_REQUESTED** [review]
@Hikkigaya please input the reference to the new Jira Data Center page to `integrations/jira/index.adoc`.

**Comment by tungmhoang** [comment]
@erincbailey you can preview the content now: http://10.2.7.208/integrations/jira/enable-jira-data-center-integration

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Select the **System** tab.
+
+Copy the value of **Base URL** and note it down
+
+image::integrations:integration-jira-enable-jira-dc-integration-base-url-administration.PNG[width=700,alt="Going to the systems tab and copying the Base URL to note it down or save it"]
+
+== Enable Jira Data Center integration
+
+Log into the Kobiton portal with an account with ADMIN predefined role.
```
Comment (erincbailey) [inline docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:64]: Log in to the Kobiton portal with an account with the ADMIN predefined role.

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Select **Connect**.
+
+If you see an error, revise the connection information, and double-check the network between the Kobiton portal and Jira Data Center.
+
+A new browser tab opens. Allow Kobiton the permissions to access Jira Data Center.
+
+image::integrations:integration-jira-enable-jira-dc-integration-access-data-center-permissions.PNG[width=500,alt="Kobiton asking for permissoin to access your Jira account"]
+
+If you see the success message, your organization is now integrated with Jira Data Center. The Jira Data Center users can follow this guide to xref:jira/create-a-jira-ticket.adoc[create Jira tickets].
```
Comment (erincbailey) [inline docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:86]: Change "The Jira Data Center users" to "Your Jira Data Center users"

**Review by erincbailey — CHANGES_REQUESTED** [review]
Please see my comments in new docs. Also - this feature impacts several other pages. Please review the below comments:
https://docs.kobiton.com/integrations/jira/
- Change page title Enable Jira integration to Enable Jira Cloud integration
- Change “session overview” to “Session Explorer” - do not put a link. It can be done from both overview and explorer right now. Plus, this will be changing with the Session Explorer redesign - so keep it simple here for now.
- Add the new page "Enable Jira Data Center integration" with a link as the second bullet under "In this section"

Left hand Nav: 
- Change Enable Jira integration to Enable Jira Cloud integration

https://docs.kobiton.com/integrations/jira/enable-jira-integration
- Change title to Enable Jira Cloud integration
- Change first sentence “Learn how to enable Jira Cloud”
- In the Note - is that Kobiton account owners or Jira account owners or both? Please clarify
- Need new screenshot at the bottom that shows there are two options. Show Jira Cloud selected. Add verbiage about selecting the Jira Cloud tab as is done in the new Jira Data Center page.

/integrations/jira/create-a-jira-ticket
- Change link for “Jira has been enabled for your organization to the overview since there are two options now - https://docs.kobiton.com/integrations/jira/
- The Jira ticket button is in both Session Overview and Session Explorer. So, update the line to say: “From Session Overview or Session Explorer, select Jira ticket > Create.”
- After “Fill out the Project, Issue Type, and Title.” add “A link to the session and other session details are prepopulated in the Description."


http://10.2.7.208/session-explorer/manage-sessions#_create_a_jira_ticket
- Jira Ticket should be lowercase ticket
- Link “Jira ticket” to /integrations/jira/create-a-jira-ticket

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
+= Enable Jira Data Center integration
+:navtitle: Enable Jira Data Center integration
+
+Learn how to integrate with Jira Data Center so your users can create Jira tickets right from the xref:session-explorer:manage-sessions.adoc[session overview].
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:4]: For the time being, appended the to the point link `_create_a_jira_ticket` and changed "session overview" to "Session Explorer"

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+Learn how to integrate with Jira Data Center so your users can create Jira tickets right from the xref:session-explorer:manage-sessions.adoc[session overview].
+
+== Before you start
+
+* Set up a Jira Data Center instance in the cloud or on-premise.
+* For Cloud and Hybrid customers, ensure that the Jira Data Center can connect to the Kobiton portal. The most straightforward way is to host Jira Data Center on a public server. If this is not an option, additional instructions will come soon.
+* For Standalone/On-prem customers, instructions to connect to Jira Data Center will come soon.
+* An admin account on Jira Data Center and an account with ADMIN predefined role in Kobiton.
+* Subscribe your Kobiton account to a plan with the Jira Integration item.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:12]: Changed.

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+== Create application links
+
+Log into Jira Data Center as an admin.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:16]: Changed.

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+Select **Save** when done.
+
+The Credentials page displays.
+
+image::integrations:integration-jira-enable-jira-dc-integration-credentials.PNG[]
+
+[TIP]
+===============================
+You can always revisit the Credentials page by select the **more** icon of the link in the Application links list and choose **View credentials**.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:48]: Word changed, and Note moved below the next step.

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+image::integrations:integration-jira-enable-jira-dc-integration-tip-application-links.PNG[width=600,alt="Choosing View credentials by clicking the more icon"]
+
+===============================
+
+Copy **Client ID** and **Client secret** and note them down.
+
+Select the **System** tab.
+
+Copy the value of **Base URL** and note it down
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:58]: Added

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc (hunk 1 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:64]: into changed to in to

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc (hunk 2 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:86]: Changed.

**Comment by Hikkigaya** [comment]
The following have also been addressed:

Please see my comments in new docs. Also - this feature impacts several other pages. Please review the below comments:
https://docs.kobiton.com/integrations/jira/

- Change page title Enable Jira integration to Enable Jira Cloud integration
- Change “session overview” to “Session Explorer” - do not put a link. It can be done from both overview and explorer right now. Plus, this will be changing with the Session Explorer redesign - so keep it simple here for now.
- Add the new page "Enable Jira Data Center integration" with a link as the second bullet under "In this section"

Left hand Nav:
- Change Enable Jira integration to Enable Jira Cloud integration

https://docs.kobiton.com/integrations/jira/enable-jira-integration

- Change title to Enable Jira Cloud integration
- Change first sentence “Learn how to enable Jira Cloud”

**Comment by Hikkigaya** [comment]
The following 3 changes require input from Tung:

https://docs.kobiton.com/integrations/jira/enable-jira-integration 

- In the Note - is that Kobiton account owners or Jira account owners or both? Please clarify
- Need new screenshot at the bottom that shows there are two options. Show Jira Cloud selected. Add verbiage about selecting the Jira Cloud tab as is done in the new Jira Data Center page.

/integrations/jira/create-a-jira-ticket

- Change link for “Jira has been enabled for your organization to the overview since there are two options now - https://docs.kobiton.com/integrations/jira/

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 8]
```diff
@@ -0,0 +1,92 @@
+= Enable Jira Data Center integration
+:navtitle: Enable Jira Data Center integration
+
+Learn how to integrate with Jira Data Center so your users can create Jira tickets right from the xref:session-explorer:manage-sessions.adoc[session overview].
+
+== Before you start
+
+* Set up a Jira Data Center instance in the cloud or on-premise.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:8]: @tungmhoang

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc [hunk 9]
```diff
@@ -0,0 +1,92 @@
+= Enable Jira Data Center integration
+:navtitle: Enable Jira Data Center integration
+
+Learn how to integrate with Jira Data Center so your users can create Jira tickets right from the xref:session-explorer:manage-sessions.adoc[session overview].
+
+== Before you start
+
+* Set up a Jira Data Center instance in the cloud or on-premise.
+* For Cloud and Hybrid customers, ensure that the Jira Data Center can connect to the Kobiton portal. The most straightforward way is to host Jira Data Center on a public server. If this is not an option, additional instructions will come soon.
```
[the remark this answers is not in the document]
Comment (Hikkigaya) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:9]: @tungmhoang

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc (hunk 8 above)
[the remark this answers is not in the document]
Comment (tungmhoang) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc]: updated.

File: docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc (hunk 9 above)
[the remark this answers is not in the document]
Comment (tungmhoang) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc:8]: Let's check with our marketing team. I prefer capitalize the first letter (Kobiton Portal)

[the remark this answers is not in the document]
Comment (tungmhoang) [reply docs/modules/integrations/pages/jira/enable-jira-data-center-integration.adoc]: Updated to Kobiton Portal

**Comment by tungmhoang** [comment]
@erincbailey All comments have been addressed. Preview link: http://10.2.7.208/integrations/jira/

**Review by tungmhoang — APPROVED** [review]

File: docs/modules/integrations/pages/jira/create-a-jira-ticket.adoc [hunk 10]
```diff
@@ -1,28 +1,28 @@
...
+
+Select a session.
+
+image::session-explorer:session-search-context.png[width=1000,alt="A context of Session page in Kobiton portal"]
+
+From **Session Overview** or **Session Explorer**, select **Jira ticket** -> **Create**.
+
+image::integrations:jira-create-ticket-button.png[width="500",alt="Select Jira ticket and then Create in Session Overview"]
+
+Fill out the *Project*, *Issue Type*, and *Title*. A link to the session and other session details are prepopulated in the Description.
```
Comment (erincbailey) [inline docs/modules/integrations/pages/jira/create-a-jira-ticket.adoc:24]: If they are not signed into Jira - will they get a modal to log in?

File: docs/modules/integrations/pages/jira/enable-jira-integration.adoc [hunk 11]
```diff
@@ -1,31 +1,32 @@
...
+
+== Connect your account
+
+Log into the Kobiton Portal.
+
+include::profile:partial$open-settings.adoc[]
+
+Select *Integration*. Under *Issue Tracking*, locate the Jira integration and select *Connect Account*.
+
+image::integrations:integration-jira-enable-jira-dc-integration-kobiton-connect-account.PNG[width=1000,alt="A closeup image of the integrations page with Jira highlighted."]
```
Comment (erincbailey) [inline docs/modules/integrations/pages/jira/enable-jira-integration.adoc:28]: This image is left-justified. Others are centered. Need to be consistent. It looks like we should be left-justifying if I look at other pages. Please confirm and let's note a standard somewhere! May need to update other Jira pages.

**Review by erincbailey — CHANGES_REQUESTED** [review]
Couple questions/comments.

File: docs/modules/integrations/pages/jira/create-a-jira-ticket.adoc (hunk 10 above)
Comment (tungmhoang) [reply docs/modules/integrations/pages/jira/create-a-jira-ticket.adoc:30]: Yes, I have updated this point.

File: docs/modules/integrations/pages/jira/enable-jira-integration.adoc (hunk 11 above)
Comment (tungmhoang) [reply docs/modules/integrations/pages/jira/enable-jira-integration.adoc]: Changed to left-justified.

**Comment by tungmhoang** [comment]
@erincbailey all feedbacks addressed.

**Comment by tungmhoang** [comment]
Since Erin is OOO, can you help to review instead @frankwmoyer ?

**Review by erincbailey — APPROVED** [review]
thanks!

**Pull request merged**